dis_snek/models/discord/guild.py

```diff
@@ -209,57 +195,57 @@ def _process_dict(cls, data, client):
         return data
 
     @property
-    def channels(self) -> List["TYPE_GUILD_CHANNEL"]:
+    def channels(self) -> List["models.TYPE_GUILD_CHANNEL"]:
```

Comment:
That would make it try and import that file during initialisation, causing circular imports

Reply:
actually, what's the condition for putting string typing? (apart from when its a class) I think I still see a few instance of it being inconsistent

Reply:
> That would make it try and import that file during initialisation, causing circular imports

oh rightttt

Reply:
It's not a string when there is a reasonable chance that it *wont* cause circular imports. IE you bring up `Snowflake_Type`, which now doesnt depend on any other major model, meaning it can be outside of a string, and imported directly